contracts/JuiceBuybackDelegate.sol

```diff
@@ -392,7 +392,7 @@ contract JuiceBuyback is IJBFundingCycleDataSource, IJBPayDelegate, IUniswapV3Sw
 
   */
   function setReservedRateOf(uint256 _projectId, uint16 _reservedRate) external onlyOwner {
-    if(_reservedRate > JBConstants.MAX_RESERVED_RATE) revert JuiceBuyback_InvalidReservedRate();
+    if(_reservedRate >= JBConstants.MAX_RESERVED_RATE) revert JuiceBuyback_InvalidReservedRate();
```

Comment:
Shouldn't revert on ==, rather shortcut it (I'll update in the other branch so we keep this one for tests)